etc/config-template.toml

```diff
@@ -158,6 +158,18 @@
 ## Otherwise, the OOM problem might occur in TiKV.
 # capacity = "1GB"
 
+[storage.gc]
+## The threshold of redundant ratio in a region to trigger GC.
+## The default value 1.1 should work for most cases. Set it to a value less than 1.0 to always trigger GC.
+# ratio-threshold = 1.1
+
+## The number of keys to GC in one batch.
+# batch-size = 512
+
+## Max bytes that GC worker can write to rocksdb in one second.
+## If it is set to 0, there is no limit.
+# max-write-bytes-per-sec = "5MB"
```

Comment:
I prefer if this can be set without restating TiKV

Reply:
Of course it can be. Maybe I can do it in a new PR.

Reply:
Do we need a `max-write-keys-per-sec`?

Reply:
No. I think `max-write-bytes-per-sec` is enough.